mona_sdk/client_util.py

```diff
@@ -17,3 +21,14 @@ def remove_items_by_value(data, value_to_remove):
     value.
     """
     return {key: value for key, value in data.items() if value != value_to_remove}
+
+
+def calculate_normalized_hash(context_id):
+    """
+    Calculate a normalized hash of a context id (a fraction between 0 to 1).
+    """
+    context_id_encoded = context_id.encode("ascii")
+    return (
+        int(hashlib.sha224(context_id_encoded).hexdigest(), base=16)
```

Comment:
please comment on the importance of using sha224 here to have a different hashing than sha1 which is used on our end for similar purposes which shouldn't be affected